source/Applications/Advanced/PoseConversions/App.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
```

Comment:
This problem isn't introduced by this sample, but I notice we're inconsistent with these version numbers. Some samples say 4.5.2 and some say 4.6.1. I think we should decide on one single version.

Reply:
I will make a seperate PR (or seperate commit in this PR) updating them to one standard.